src/components/Features.jsx

```diff
@@ -32,118 +32,122 @@ const Container = styled.div`
 `;
 
 const DecorationContainer = styled.div`
-display: flex;
-flex-wrap: wrap;
-gap: 2rem;
-justify-content: center;
+  display: flex;
+  flex-wrap: wrap;
+  gap: 2rem;
+  justify-content: center;
 `;
 
 const DecorationData = styled.div`
-text-align: center;
-padding: 1rem 1rem 2rem;
-background-color: var(--container-color);
-box-shadow: 0 2px 6px rgba(65, 11, 16, 0.15);
-border-radius: 1rem;
-width: calc(25% - 1rem);
-max-width: 220px;
-flex: 1 0 auto;
-box-sizing: border-box;
+  text-align: center;
+  padding: 2rem;
+  background: rgba(255, 255, 255, 0.1);
+  box-shadow: 0 2px 6px rgba(65, 11, 16, 0.15);
+  backdrop-filter: blur(10px);
+  border-radius: 1rem;
+  width: calc(25% - 1rem);
+  max-width: 250px;
+  flex: 1 0 auto;
```

Comment:
redundant.  you have written `display:flex` again in line 52. Choose which to keep and remove.